test/test_common/utility.cc

```diff
@@ -347,7 +347,11 @@ MockedTestAllocator::~MockedTestAllocator() {}
 namespace Thread {
 
 ThreadFactory& threadFactoryForTest() {
-  static ThreadFactoryImpl* thread_factory = new ThreadFactoryImpl();
+#if !defined(WIN32)
```

Comment:
flip if/else and invert the conditional

Reply:
So far, we've been following this pattern (`if !defined(WIN32)`) in our PRs. We don't have a strong opinion on which way the conditional should be written, but do think we should be consistent about which way we pick.

There's about ~8 other places we'd have to change as well, all but 1 in test code. If we want to make the change as part of this PR, that's fine with us.